src/GrumPHP/Locator/ChangedFiles.php

```diff
@@ -41,10 +41,23 @@ public function __construct(Repository $repository, Filesystem $filesystem)
     public function locateFromGitRepository()
     {
         $diff = $this->repository->getWorkingCopy()->getDiffStaged();
-
         return $this->parseFilesFromDiff($diff);
     }
+    
+    /**
+     * @return FilesCollection
+     */
+    public function locateFromGitPushedRepository()
+    {
+        $diff = explode("\n", \Gitonomy\Git\Repository::run('diff origin/master..HEAD --name-only --oneline'));
```

Comment:
If there is one registered remote, we might just use that one? On multiple, we won't know the remote. This PR is getting a bit complex :)